hierarchy-tool/tool/src/webapp/WEB-INF/velocity/show.vm

```diff
@@ -3,91 +3,217 @@
 
 #chef_portletContainer()
 #chef_portletContent()
+<style>
+    table, th, td {
+        border-collapse: collapse;
+    }
+    th, td {
+        padding: 5px;
+        text-align: left;
+        vertical-align:bottom;
+    }
+    .label_h3 {
+        color: #555;
```

Comment:
Same goes for colors as fonts, generally they should come from tool.css so that they can be controlled by the skin.